src/styles/footer/footer.css

```diff
@@ -68,13 +73,17 @@
   flex: auto;
 }
 
-.newsletter__link,
-.newsletter__link image {
+.newsletter__link {
   transition: 200ms;
 }
 
-.newsletter__link:hover image,
-.newsletter__link:focus image {
+.newsletter__link svg {
+  height: 1.5rem;
+  width: 1.5rem;
+}
+
+.newsletter__link:hover,
+.newsletter__link:focus {
```

Comment:
this will increase the whole link element, I guess we should only increase svg size